test/integration/main_test.go

```diff
@@ -177,6 +186,10 @@ contacts:
 				t.Errorf("resourcemapping(ignore annotation) unexpected diff (-want,+got):\n%s", diff)
 			}
 
+			if diff := cmp.Diff(wantGithubResource, gotPmapEvent.GetGithubSource(), cmpopts.IgnoreUnexported(v1alpha1.GitHubSource{})); diff != "" {
```

Comment:
Do we have any unexported fileds in `GitHubSource`?

Reply:
Yes, https://github.com/abcxyz/pmap/blob/add_pmap_event_info/apis/v1alpha1/pmap_event.pb.go#L116-L119